test/e2e/app-dir/dynamic-io/dynamic-io.params.test.ts

```diff
@@ -2025,8 +2025,6 @@ describe('dynamic-io', () => {
             expect.stringContaining(
               'enumerated with `{...params}`, `Object.keys(params)`, or similar.'
             ),
-            expect.stringContaining('accessed directly with `params.lowcard`'),
-            expect.stringContaining('accessed directly with `params.highcard`'),
```

Comment:
This is an interesting case. If you spread the params into a new object and then later access the access a specific value from that new object, we won't warn again. We could restore the old behavior if we key by frame and type of access.

If we don't, we risk having a whack-a-mole scenario where you stop spreading to fix one warning only to get new warnings afterwards. I assume here that the fix is awaiting the spread which is also what the codemod would do i.e. deduping this is fine.